homeassistant/components/stream/worker.py

```diff
@@ -29,7 +29,15 @@ def create_stream_buffer(stream_output, video_stream, audio_frame):
 
     a_packet = None
     segment = io.BytesIO()
-    output = av.open(segment, mode="w", format=stream_output.format)
+    output = av.open(
+        segment,
+        mode="w",
+        format=stream_output.format,
+        container_options={
+            "video_track_timescale": str(int(1 / video_stream.time_base)),
```

Comment:
This was important. The video_track_timescale didn't seem to get set correctly from video_stream template without it (it was using the base framerate and then calculating a timescale based off that (https://github.com/FFmpeg/FFmpeg/blob/d5247fb1da615a7ccff9fa7271bc8aba23e0de25/libavformat/movenc.c#L6498). Since we are not reencoding our streams, we should just use the timescale of the input stream.